backend/python/transformers/transformers_server.py

```diff
@@ -75,17 +75,34 @@ def LoadModel(self, request, context):
             A Result object that contains the result of the LoadModel operation.
         """
         model_name = request.Model
+
+        compute = torch.float32
+        if request.F16Memory == True:
+            compute=torch.bfloat16
+
+        self.CUDA = request.CUDA
+
+        device_map="cpu"
+
+        quantization = BitsAndBytesConfig(
```

Comment:
as we do actually have a `quantization` field in the gRPC requests and BitsAndBytes is one of the available ( see https://huggingface.co/docs/transformers/main_classes/quantization#quantization), we might want to select `BitsAndBytesConfig` when setting `quantization=BitsAndBytesConfig` instead? so we keep the backward compatibility change too.

Reply:
Thank you @mudler I think it's a good point.

What do you think of using something like:

- `quantization=bnb_4bit`
- `quantization=bnb_8bit`

to select between the two options?

Reply:
@fakezeta that looks reasonable to me :+1: